src/aleph/services/cost.py

```diff
@@ -126,10 +131,20 @@ def _get_compute_unit_multiplier(content: ExecutableContent) -> int:
 
 def compute_cost(session: DbSession, content: ExecutableContent) -> Decimal:
     is_on_demand = isinstance(content, ProgramContent) and not content.on.persistent
+    is_confidential_vm = (
+        not is_on_demand
+        and isinstance(getattr(content, "environment", None), InstanceEnvironment)
+        and getattr(content.environment, "trusted_execution", False)
+    )
+
```

Comment:
Refactoring this variable assignment, and the next one, into either 2 if/else or match/case block would probably increase readability instead of nesting ternary operators